### PR TITLE
Feature/add inline image gallery and horizontal pager

### PR DESCRIPTION
**Pull request by ikalafat**
- adding inline gallery docs
- adding image gallery docs
- adding horizontal pager docs

### CODE REVIEW

File: _posts/1-2-28-InlineGallery.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+layout: doc
+permalink: /docs/ui-toolkit/components/inline-gallery
+title: Inline Gallery
+section: UI toolkit
+---
+
+# InlineGallery
+
+ImageGallery component renders a collection of `Image` components within `HorizontalPager`. Each preview component is rendered on a separate page.
```
Comment (tenodi) [inline _posts/1-2-28-InlineGallery.md]: Inline gallery?

File: _posts/1-2-28-InlineGallery.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## InlineGallery
+![Inline Gallery example (left image)]({{ site.baseurl }}/img/ui-toolkit/image-gallery/image_gallery@2x.png "InlineGallery"){:.docs-component-image}
```
Comment (tenodi) [inline _posts/1-2-28-InlineGallery.md]: We have the same picture as for image gallery?

File: _posts/1-2-28-InlineGallery.md (hunk 1 above)
Comment (ikalafat) [inline _posts/1-2-28-InlineGallery.md]: Fixed. thanks

File: _posts/1-2-28-InlineGallery.md (hunk 2 above)
Comment (ikalafat) [inline _posts/1-2-28-InlineGallery.md]: Yeah, we don't have splitted images.. similar for horizontal pager / page indicators

File: _posts/1-2-29-PageIndicators.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+title: Page Indicators
+section: UI toolkit
+---
+
+# PageIndicators
+
+Renders a horizontally aligned list of dots (page indicators), which can be useful in `HorizontalPager`, `ImageGallery` or `InlineGallery` components
+
+## PageIndicators
+![PageIndicators example]({{ site.baseurl }}/img/ui-toolkit/horizontal-pager/horizontal_pager@2x.png "PageIndicators"){:.docs-component-image}
```
Comment (tenodi) [inline _posts/1-2-29-PageIndicators.md]: This picture is good, although it should be called page_indicators. Also, it's a bit confusing that the same picture is used in `HorizontalPager`. If I use `HorizontalPager`, but I'm not getting these dots, we shouldn't use that image. Instead, we could provide an example image of using HorizontalPager + PageIndicators + some content that I'm actually scrolling.

File: _posts/1-2-28-InlineGallery.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
+---
+layout: doc
+permalink: /docs/ui-toolkit/components/inline-gallery
+title: Inline Gallery
+section: UI toolkit
+---
+
+# InlineGallery
+
+InlineGallery component renders a collection of `Image` components within `HorizontalPager`. Each preview component is rendered on a separate page.
```
Comment (tenodi) [inline _posts/1-2-28-InlineGallery.md]: Based on image and description, I don't understand what's the difference between `ImageGallery` and `InlineGallery`.

**Pull request closed without merging**